src/includes/classes/Plugin.php

```diff
@@ -338,7 +348,10 @@ public function setup()
             'auto_cache_delay'       => '500', // In milliseconds.
             'auto_cache_sitemap_url' => 'sitemap.xml', // Relative to `site_url()`.
             'auto_cache_other_urls'  => '', // A line-delimited list of any other URLs.
-            'auto_cache_user_agent'  => 'WordPress',
+            'auto_cache_permalinks'  => '1', // `0|1`; auto-cache known Post/Page permalinks?
```

Comment:
> ha.. I am actually using the term sonic boom when I talk about ZenCache now.

We should totally work that into the UI somehow... "**Enable (safest)** | **Enable (fastest)** | **Enable (sonic boom!)**"